apis/validator/beacon_committee_subscriptions.v2.yaml

```diff
@@ -0,0 +1,61 @@
+post:
+  operationId: "prepareBeaconCommitteeSubnetV2"
+  summary: Signal beacon node to prepare for a committee subnet
+  description: |
+    After beacon node receives this request:
+    - Search using discv5 for peers related to this subnet and replace current peers with those ones if necessary. 
+    - Check each provided `slot_signature`, to determine whether any validator has been [assigned the duty of aggregation](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/validator.md#aggregation-selection)
```

Comment:
The summary and description probably need to be updated to reflect the new flow. VCs do not "signal" the BN anymore, but rather "VC provides data to maybe signal and BN responds with wether it was signalled".